### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ZakiChammaa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -60,3 +60,7 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+#files
+temp_config.py
```
Comment (justinjeanpierre) [inline .gitignore]: I've added the config file locally as Config.py.  You can/should delete temp_config.py if you are no longer using it.

File: simon.py [hunk 2]
```diff
@@ -92,6 +81,105 @@ def oauthorized():
...
+    resp = facebook.authorized_response()
+    if resp is None:
+        return 'Access denied: reason=%s error=%s' % (
+            request.args['error_reason'],
+            request.args['error_description']
+        )
+    if isinstance(resp, OAuthException):
+        return 'Access denied: %s' % resp.message
+
+    session['oauth_token'] = (resp['access_token'], '')
```
Comment (justinjeanpierre) [inline simon.py]: Wouldn't 'facebook_token' be a better name for this session key?

File: templates/simulation.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{% block content %}
+	<div class="row">
+		<div class="col-md-5">
+				<h2 class="page-header">{% block title %}Run Simulation{% endblock %}</h2>
+				<form method="post" class="form-horizontal" action="/jobs">
+	            	<div class="form-group">
+	                	<div class="col-sm-2"></div>
+	            	</div>
+	            	<label for="nc">Number of Cores:</label>
+	            	<input type="text" class="form-control" name="cores" id = "nc" value="{{request.form.cores}}">
```
Comment (justinjeanpierre) [inline templates/simulation.html]: I think the names here should be the same as in /data/default_simulator_parameters.plist.

**Comment by justinjeanpierre** [comment]
Please merge [PR 12](https://github.com/justinjeanpierre/simon/pull/12) first.

**Comment by justinjeanpierre** [comment]
I'm ready to merge this once these changes are finished.

File: .gitignore [hunk 4]
```diff
@@ -60,3 +60,10 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+<<<<<<< HEAD
```
Comment (justinjeanpierre) [inline .gitignore]: These probably shouldn't be here.

Comment (ZakiChammaa) [inline .gitignore]: I dunno why they appeaed, I removed them manually from simon

**Pull request merged**